internal/model/booking.go

```diff
@@ -26,3 +28,42 @@ func (b *Booking) Update(db *gorm.DB) error {
 func (b *Booking) Delete(db *gorm.DB) error {
 	return db.Delete(b).Error
 }
+
+// populate bookings table with sample values
+
+func (*Booking) PopulateBookings(db *gorm.DB) {
```

Comment:
(applies to other Populate methods)

what is the reasoning behind using an unnamed parameter here?

https://stackoverflow.com/questions/40950877/is-unnamed-arguments-a-thing-in-go

based on this it seems to be for satisfying interfaces

Reply:
i've changed it to a function instead of a variable! 
i needed a parameter because it was a method and so i needed an instance to call it even though it's not being used :grinning: